### PR TITLE
CFY-7845 update plugin upload

### PR DESCRIPTION
**Pull request by dusking**

### CODE REVIEW

File: content/plugins/overview.md [hunk 1]
```diff
@@ -56,8 +56,9 @@ _Note: This method is available when working in a [Local CLI profile]({{< relref
 
 Plugin usage inside of blueprints varies. However, these two general rules apply:
 
-- Your blueprint must import the `plugin.yaml` of the pluigns you would like to use. See [importing]({{< relref "blueprints/spec-imports.md" >}}) plugins for more information. See also [spec-plugins]({{< relref "blueprints/spec-plugins.md" >}}).
-- Either the `plugin.yaml` or your blueprint will map node lifecycle operations to appropriate plugin functions.
+- Your blueprint must add import statement for the plugins that in use. See [importing]({{< relref "blueprints/spec-imports.md" >}}) plugins for more information
+- We still support importing the `plugin.yaml` of the pluigns you would like to use. See [spec-plugins]({{< relref "blueprints/spec-plugins.md" >}}).
```
Comment (yarivgiga) [inline content/plugins/overview.md]: little typo in plugins...

File: content/blueprints/spec-imports.md [hunk 2]
```diff
@@ -47,3 +47,30 @@ A few important things to know about importing YAML files:
 * An error is returned if there are cyclic imports (i.e. a file is importing itself or you are attempting to import a file that is importing the file that imported it, etc..)
 * [Groups]({{< relref "blueprints/spec-groups.md" >}}) cannot be imported and can only be defined in the main blueprint file
 * The `tosca_definitions_version` as stated [here]({{< relref "blueprints/spec-versioning.md" >}}) must match across imported files.
+
+
+# Example
+
+{{< gsHighlight  yaml >}}
```
Comment (yarivgiga) [inline content/blueprints/spec-imports.md]: what is that?

File: content/blueprints/spec-imports.md [hunk 3]
```diff
@@ -47,3 +47,30 @@ A few important things to know about importing YAML files:
...
+
+node_templates:
+  vm:
+    type: cloudify.openstack.nodes.Server
+  webserver:
+    type: cloudify.nodes.WebServer
+{{< /gsHighlight >}}
+
+
+In the above example, instead of importing the plugin.yaml file, we declare that we're using the plugins.
```
Comment (bmansheim) [inline content/blueprints/spec-imports.md]: we're > we are
In general better not to use contractions

File: content/blueprints/spec-imports.md [hunk 4]
```diff
@@ -47,3 +47,30 @@ A few important things to know about importing YAML files:
...
+This is the best-practice for using plugins inside blueprint.
```
Comment (bmansheim) [inline content/blueprints/spec-imports.md]: best-practice > best practice

File: content/blueprints/spec-imports.md [hunk 5]
```diff
@@ -47,3 +47,30 @@ A few important things to know about importing YAML files:
...
+
+The plugin import format is: `plugin:PLUGIN_NAME?version=VERSION&distribution=DISTRIBUTION`
+There parameters are optional, and are aimed to resolve cases when the managers have multiple similar plugins, with the same name.
```
Comment (bmansheim) [inline content/blueprints/spec-imports.md]: Remove commas

File: content/cli/plugins.md [hunk 6]
```diff
@@ -13,6 +13,8 @@ You can use the command to upload, download, delete and list plugins and also to
 
 A Cloudify plugin is an archive created by [wagon](http://github.com/cloudify-cosmo/wagon).
 
+Each plugin has a plugin.yaml file, which map node lifecycle operations to appropriate plugin functions.
```
Comment (bmansheim) [inline content/cli/plugins.md]: file, which map > file that maps

File: content/cli/plugins.md [hunk 7]
```diff
@@ -37,6 +39,10 @@ Upload a plugin to Cloudify Manager.
 Wagon (via the `--format` flag) enables you to create archives in both `tar.gz` and `zip` formats. Cloudify only supports wagon in the `tar.gz` format.
 {{% /gsNote %}}
 
+#### Required flags
+
+* `-y, --yaml-path TEXT` - The path to the plugin's yaml file
```
Comment (bmansheim) [inline content/cli/plugins.md]: plugin's yaml file > yaml file for the plugin

File: content/plugins/overview.md [hunk 8]
```diff
@@ -56,8 +56,9 @@ _Note: This method is available when working in a [Local CLI profile]({{< relref
 
 Plugin usage inside of blueprints varies. However, these two general rules apply:
 
-- Your blueprint must import the `plugin.yaml` of the pluigns you would like to use. See [importing]({{< relref "blueprints/spec-imports.md" >}}) plugins for more information. See also [spec-plugins]({{< relref "blueprints/spec-plugins.md" >}}).
-- Either the `plugin.yaml` or your blueprint will map node lifecycle operations to appropriate plugin functions.
+- Your blueprint must add import statement for the plugins that in use. See [importing]({{< relref "blueprints/spec-imports.md" >}}) plugins for more information
```
Comment (bmansheim) [inline content/plugins/overview.md]: add import statement for the plugins that in use > add an import statement for the plugins that are in use
In general, we don't remove words that seem unecessary

File: content/plugins/overview.md [hunk 9]
```diff
@@ -56,8 +56,9 @@ _Note: This method is available when working in a [Local CLI profile]({{< relref
...
+- We still support importing the `plugin.yaml` of the plugins you would like to use. See [spec-plugins]({{< relref "blueprints/spec-plugins.md" >}}).
```
Comment (bmansheim) [inline content/plugins/overview.md]: you would like to use > you want to use
We want to be more direct than polite

**Pull request merged**